website/server/models/user/hooks.js

```diff
@@ -45,6 +45,13 @@ function _populateDefaultTasks (user, taskTypes) {
     taskTypes.splice(tagsI, 1);
   }
 
+  function findTag (tagName) {
```

Comment:
can this function be defined at the root level of the file instead of inside another function?